lib/div-alg.gi

```diff
@@ -1958,7 +1900,7 @@ for i in [1..k] do
     x:=Filtered([1..k],j-> e[i,j] mod Gcd(m,hrs[j][2]-1) = 0);
     if Length(x) > 0 then
       as:=[];
-      for j in [1..k] do
+      for j in Difference([1..k],[i]) do
```

Comment:
Another merge conflict because of merging #78 was here. I hope I resolved it right too.